Classes/Command/EventStoreCommandController.php

```diff
@@ -37,118 +39,102 @@ class EventStoreCommandController extends CommandController
-     * parameters which were defined for Flow.
+     * This command initializes the given Event Store adapters (i.e. creates required tables if it is database
+     * driven and/or validates the configuration against the actual backend)
      *
+     * @param string $eventStore The identifier of the Event Store to set up
      * @return void
      */
-    public function createSchemaCommand()
+    public function setupCommand($eventStore)
     {
-        $this->outputLine('Creating Event Store database tables in database "%s" on host %s connecting with user "%s" ...', [ $this->configuration['backendOptions']['dbname'], $this->configuration['backendOptions']['host'], $this->configuration['backendOptions']['user']]);
-        try {
-            $connection = $this->connectionFactory->get();
-
-            if ($connection->getSchemaManager()->tablesExist([$this->connectionFactory->getStreamTableName()])) {
-                $this->outputLine('The table %s already exists, not changing anything.', [$this->connectionFactory->getStreamTableName()]);
-                $this->quit(1);
-                exit;
-            }
-
-            $schema = $connection->getSchemaManager()->createSchema();
-            $toSchema = clone $schema;
-
-            EventStoreSchema::createStream($toSchema, $this->connectionFactory->getStreamTableName());
-
-            $connection->beginTransaction();
-            $statements = $schema->getMigrateToSql($toSchema, $connection->getDatabasePlatform());
-            foreach ($statements as $statement) {
-                $this->outputLine('<info>++</info> %s', [$statement]);
-                $connection->exec($statement);
-            }
-            $connection->commit();
-
-            $this->outputLine();
-        } catch (ConnectionException $exception) {
-            $this->outputLine('<error>Connection failed</error>');
-            $this->outputLine('%s', [ $exception->getMessage() ]);
+        $eventStores = $this->eventStoreManager->getAllEventStores();
+        if (!isset($eventStores[$eventStore])) {
+            $this->outputLine('<error>There is no Event Store "%s" configured</error>', [$eventStore]);
             $this->quit(1);
         }
+        $this->outputLine('Setting up Event Store "%s"', [$eventStore]);
+        $result = $eventStores[$eventStore]->setup();
+        $this->renderResult($result);
     }
 
     /**
-     * Drop Event Store database tables
+     * Sets up all configured Event Store backends
      *
-     * This command <b>deletes all</b> Event Store related database tables! It uses the Doctrine connection
-     * parameters which were defined for Flow.
+     * This command initializes all configured Event Store adapters (i.e. creates required tables for database
+     * driven storages and/or validates the configuration against the actual backends)
      *
      * @return void
      */
-    public function dropSchemaCommand()
+    public function setupAllCommand()
```

Comment:
Yes, I actually thought the same. I just disliked the mix between an actual Event Store identifier and the "magic" string `all`. As you wrote: can be adjusted in a follow-up